packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -125,6 +129,7 @@ const metadata = {
   showIncomingTransactions: { persist: true, anonymous: true },
   useNftDetection: { persist: true, anonymous: true },
   useTokenDetection: { persist: true, anonymous: true },
+  smartTransactionsOptInStatus: { persist: true, anonymous: true },
```

Comment:
Curious: does this need to be anonymous? setting it to `false` will allow us to receive this parameter in the debug snapshot

Reply:
I set this to `false` to enable better debugging, thank you for the suggestion